tests/tlsconfig_test.go

```diff
@@ -120,6 +121,63 @@ var _ = Describe("TLS config", func() {
 				Expect(connState.Version).To(BeNumerically(">=", tls.VersionTLS10))
 			}, 1*time.Minute, time.Second).Should(Succeed())
 		})
+
+		It("should reload config if file recreated after deletion", func() {
+			Eventually(func() error {
+				foundConfig, err := ApiClient.CoreV1().ConfigMaps(DeploymentNamespace).Get(context.TODO(), configMapName, metav1.GetOptions{})
+				if err != nil {
+					return err
+				}
+
+				delete(foundConfig.Data, console.TlsProfileFile)
+
+				_, err = ApiClient.CoreV1().ConfigMaps(DeploymentNamespace).Update(context.TODO(), foundConfig, metav1.UpdateOptions{})
+				return err
+			}, 1*time.Minute, time.Second).Should(Succeed())
+
+			// Wait until default values take effect
+			Eventually(func(g Gomega) {
+				connState, err := getTlsConnectionState()
+				Expect(err).ToNot(HaveOccurred())
+				Expect(connState.Version).To(BeNumerically(">=", tls.VersionTLS10))
+			}, 1*time.Minute, time.Second).Should(Succeed())
+
+			// Recreate file
+			tlsProfile := &api.TlsSecurityProfile{
+				Type:   ocpconfigv1.TLSProfileModernType,
```

Comment:
Can you use a custom profile so this test is independent of changes to the modern profile in the future?

Reply:
done